frontend/src/components/AddTodo.tsx

```diff
@@ -13,14 +13,15 @@ export default function AddTodo(props: Props) {
         setDescription(event.target.value)
     }
 
-    function onSave() {
-        const newTodo: NewTodo = {description, status: "OPEN"}
+    function onSave(event: any) {
+        console.log(event)
+        const newTodo: NewTodo = {description, status: "open"}
         props.onAdd(newTodo)
             .then(r => setDescription(""))
     }
 
-    return <div>
+    return <form onSubmit={onSave}>
```

Comment:
according with change from onSave to onSubmit
```suggestion
    return <form onSubmit={onSubmit}>
```